quinn-proto/src/token_store.rs

```diff
@@ -0,0 +1,18 @@
+use bytes::Bytes;
+
+/// Responsible for storing address validation tokens received from servers and retrieving them for
+/// use in subsequent connections
+pub trait TokenStore: Send + Sync {
```

Comment:
Prefer to put this in an existing module.

Reply:
Splitting off into follow-up PR